jax/_src/cudnn/fused_attention_stablehlo.py

```diff
@@ -151,6 +152,7 @@ def create_dot_product_attention_backend_config(batch,
     "seed": seed,
     "is_flash_attention": True,
     "mask_type": convert_mask_type_to_string(mask_type),
+    "sliding_window_length": sliding_window_length if not None else 0,
```

Comment:
Are we sure this is correct? `not None` is always true.

Reply:
It can be None, I just changed the sdpa to use None as default value.

Reply:
fixed.